packages/server/src/api/routes/public/index.ts

```diff
@@ -8,18 +8,46 @@ import authorized from "../../../middleware/authorized"
 import { paramResource, paramSubResource } from "../../../middleware/resourceId"
 import { CtxFn } from "./utils/Endpoint"
 import mapperMiddleware from "./middleware/mapper"
+import env from "../../../environment"
+// below imports don't have declaration files
 const Router = require("@koa/router")
+const RateLimit = require("koa2-ratelimit").RateLimit
+const Stores = require("koa2-ratelimit").Stores
```

Comment:
We can remove the duplicate imports by doing
```js
const { RateLimit, Stores } = require("koa2-ratelimit")
```
Or even better would be to use ES syntax
```js
import { RateLimit, Stores } from "koa2-ratelimit"
```
And add the types with 
```
yarn add --dev @types/koa2-ratelimit
```